src/tools/fuzzing/fuzzing.cpp

```diff
@@ -2900,7 +2905,13 @@ Type TranslateToFuzzReader::getSingleConcreteType() {
                      WeightedOption{Type::f32, VeryImportant},
                      WeightedOption{Type::f64, VeryImportant})
                 .add(FeatureSet::SIMD, WeightedOption{Type::v128, Important})
-                .add(FeatureSet::ReferenceTypes, Type::funcref, Type::anyref)
+                // Avoid Type::anyref without GC enabled, as in some contexts
+                // we will not be able to emit any constant for it (with GC, we
```

Comment:
Can't we use `ref.null`?

Reply:
In this code path I suppose null is an option, yes, I can revert this part.